### ISSUE TITLE
Gdata and GTLDrive API's don't work well together in IOS

### ISSUE THREAD

**Issue by amarzulli** [google/gdata-objectivec-client#185]
I have successfully added the GData API to my project to read and write data to spreadsheets.  Now to enable the functionality of creating a new sheet I need to use the GTLDrive API however after adding it to my project it seems to break the GData API.  I get errors stating the GData classes like GDataServiceGoogleSpreadsheet is unknown.  Am I doing something wrong when adding it to my project?  I am following the IOS Quickstart guide...https://developers.google.com/drive/ios/quickstart.

**Comment by gmrobbins** [comment]
It sounds like GDataServiceGoogleSpreadsheet.m is no longer part of your build target. The two types of APIs should be able to coexist.

**Comment by amarzulli** [comment]
Ok instead I am trying to use the Drive REST API with NSURLRequest using the same access token used for my GData requests which work.  I have authorised it for the scope https://www.googleapis.com/auth/drive.file but when trying to make a request to create a new file I am getting a 401 response.  Can I use OAuth2.0 with the Drive API and devices?  According to this site Drive scopes are not available...
https://developers.google.com/identity/protocols/OAuth2ForDevices

**Comment by amarzulli** [comment]
Solved my issue using REST calls instead.  I wasn't loading my access token correctly.  Have since tried to create a new project and add both drive and GData APIs still with no luck.  Seems now the class GTMOAuth2ViewControllerTouch is not available even though my #import works and I have made sure it is in my Compile Sources.  But I can achieve what I need to using the REST API instead.  Any idea on when the GData API will be incorporated with the Drive API?
